README.md

```diff
@@ -39,42 +39,74 @@ npx gocd-cypress --help
 
 # Configuration
 
-Configuration keys can be defined
- - in `package.json` in `gocdCypress` property:
-   ```json
-   {
-       "name": "my-project",
-       "version": "0.0.0",
-       "gocdCypress": {
-           "CY_DOCKER_IMAGE": "cypress/browsers:node16.13.0-chrome95-ff94"
-       }
-   }
-   ```
- - via environment variable:
-   ```bash
-   export CY_DOCKER_IMAGE=cypress/browsers:node16.13.0-chrome95-ff94
-   npx gocd-cypress
-   ```
+This tool uses [Cosmiconfig](https://github.com/davidtheclark/cosmiconfig) for the configuration options, see its 
+documentation about where you can define your options. The module name we use with it is `gocdCypress`.
+
+Example in `package.json`:
+```json
+{
+    "name": "my-project",
+    "version": "0.0.0",
+    "gocdCypress": {
+        "cypressCmd": "cypress run --browser firefox",
+        "dockerImage": "cypress/browsers:node16.13.0-chrome95-ff94"
+    }
```

Comment:
Nice! This and the below config option looks much better than the previous solution. 👍